### ISSUE TITLE
server3.py (utf-8 code can't decode byte 0xd6 in position 2: invalid continuation byte)

### ISSUE THREAD

**Issue by Vincent-Quant** [insidesherpa/JPMC-tech-task-1-PY3#151]
![Trouble](https://user-images.githubusercontent.com/67428752/86216427-9cf28880-bbb0-11ea-9bcc-09fc6f2f6e91.jpg)

I would like to run server3.py, but cannot continue on my task.
My system is Windows 10 (x64), and I am using python from Anaconda Navigator (The path is set up). I have checked that my python version is 3.7.6, and I sometimes I also encountered this issue when I use pd.read_csv code, but I'm not sure which part of the code of server3.py has anything to do with the problem. Please help me, thanks!

**Comment by iriyagupta** [comment]
Is this still an issue or has been resolved?

**Comment by Vincent-Quant** [comment]
It still hasn't been resolved yet. I actually got stuck after confronted with this problem.

**Comment by iriyagupta** [comment]
Does your client3.py file run without any error?

**Comment by Vincent-Quant** [comment]
No, I can not run it because I guess I am not able to use it without server3.py? 

Translation from the last line: 
Fail to connect, because the target computer refuse to make connection. 

![clientpy](https://user-images.githubusercontent.com/67428752/92329856-f12c5400-f09c-11ea-9fbe-4be1b2fd0a0e.png)

**Comment by iriyagupta** [comment]
Try changing the code where json loads in client3.py to :

 `quotes = json.loads(urllib.request.urlopen(QUERY.format(random.random())).read().decode("utf-8"))`

I am hoping the code will work after this.

**Comment by Vincent-Quant** [comment]
The above code doesn't work, but I have finally found the issue of error! It was due to the fact that the name of my computer wasn't in English, causing some error in character encoding. Anyway, still very much appreciate your kindly assistance!
